### DISCUSSION TITLE
count_private for pull requests?

### DISCUSSION THREAD

**Discussion by jpdev01** [anuraghazra/github-readme-stats#1449] [Q&A] [answered]
is there how to count the private ones? If not, can we create it easy?

**Comment by rickstaa** [comment, marked as answer]
@jpdev01 Thanks for your question. For the stats card, you can use the `count_private` flag to count the private contributions. For the pull requests and the data on the other cards, it is also possible to include data from private repositories. I have this working on my self-hosted vercel instance and in my dev environment. You can try it out by using the following query in the [graphql explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

You can then use the following in the query variables field.

```graphql
{"login": "michele-grifa"}
```

However, if you're using the public vercel instance (i.e. https://github-readme-stats.vercel.app/), it can not work for cards other than the stats card since we do not have access to your private repositories. As a result, we can not count the language bytes. I that case you have to set up your vercel instance. A guide on how this is done can be found [here](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

**Reply by rickstaa** [reply]
@jpdev01 Did that answer your question?

**Reply by jpdev01** [reply]
Yes, thanks! 🤘🏻
